platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterUtils.java

```diff
@@ -229,6 +230,34 @@ public static RosterHistory buildRosterHistory(final PlatformStateAccessor reada
         return new RosterHistory(roundRosterPairList, rosterMap);
     }
 
+    /**
+     * Determining the Roster History.
+     * There are three non-genesis modes that a node can start in:
+     * <ul>
+     *   <li> Genesis Network - The node is started with a genesis roster and no pre-existing state on disk. </li>
+     *   <li> Network Transplant - The node is started with a state on disk and an overriding roster for a different network. </li>
+     *   <li> Software Upgrade - The node is restarted with the same state on disk and a software upgrade is happening. </li>
+     *   <li> Normal Restart - The node is restarted with the same state on disk and no software upgrade is happening. </li>
+     * </ul>
```

Comment:
Useful to know, but I don't understand how this is related to the roster history?

Reply:
I agree, that became outdated and not relevant to the current implementation, updated.